.github/workflows/rust.yml

```diff
@@ -104,43 +104,48 @@ jobs:
         run: cargo test --target i686-unknown-linux-gnu
 
   Cross:
-    name: Cross test
+    name: Cross test Pending
     if: ${{ !github.event.act }}
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Crate
-        uses: actions/checkout@v4
-      - name: Checkout Toolchain
-        uses: dtolnay/rust-toolchain@stable
-      - name: Install target
-        run: rustup target add s390x-unknown-linux-gnu
-      - name: install cross
-        run: cargo install cross --locked
-      - name: run cross test
-        run: cross test --target s390x-unknown-linux-gnu
+      - name: Pending
```

Comment:
I created an issue to re-enable cross and embedded tests. https://github.com/chaintope/rust-tapyrus/issues/41